samza-core/src/main/scala/org/apache/samza/container/SamzaContainer.scala

```diff
@@ -500,8 +519,9 @@ object SamzaContainer extends Logging {
     val loggedStorageBaseDir = getLoggedStorageBaseDir(jobConfig, defaultStoreBaseDir)
     info("Got base directory for logged data stores: %s" format loggedStorageBaseDir)
 
+    // TODO dchen should we enforce restore factories to be subset of backup factories?
```

Comment:
TODO HIGH dchen. @dxichen let's fix this in the "config scope and precedence" follow up for async commit.